processing.py

```diff
@@ -18,7 +18,7 @@
 PSWAP_ID = "0x0200050000000000000000000000000000000000000000000000000000000000"
 XOR_ACCOUNT = (
     #"0x54734f90f971a02c609b2d684e61b5574e35ac9942579a2635aada58e5d836a7"  # noqa
-    "cnTQ1kbv7PBNNQrEb1tZpmK7ftiv4yCCpUQy1J2y7Y54Taiaw" #noqa
```

Comment:
is it correct? As I remember the problem was that account is decoded sometimes, but sometimes no